### ISSUE TITLE
Psych needs an x86-mingw32 binary gem

### ISSUE THREAD

**Issue by jonforums** [ruby/psych#34]
Please make binary gems available as RubyInstaller users installing or updating (with or without the DevKit installed) will almost certainly run into install failures due to 

  https://github.com/tenderlove/psych/blob/master/ext/psych/extconf.rb#L10

and not having libyaml artifacts available for a manual `--with-libyaml-dir=...` fixup.

Thanks.

**UPDATE** to clarify, RubyInstaller users can install Psych _if_ they have the DevKit installed _and_ the have they libyaml dev artifacts installed by running something similar to the following. However, I get runtime 'missing yaml.dll' error messages I'm tracking down (my `C:\devlibs\libyaml\lib` includes both `libyaml.a` and `libyaml.dll.a`)

```
C:\Users\Jon>gem install psych -- --with-libyaml-include=C:/devlibs/libyaml/include --with-lib-yaml-lib=C:/devlibs/libyaml/lib
Fetching: psych-1.2.2.gem (100%)
Temporarily enhancing PATH to include DevKit...
Building native extensions.  This could take a while...
Successfully installed psych-1.2.2
1 gem installed
```

Bottom line, manual gem install is too tweaky and a static linked binary gem would be great.

**Comment by hsbt** [comment]
I've released mingw fat gem after psych-3.0.0.

https://rubygems.org/gems/psych/versions
